scripts/init.d-attach-static-vdis

```diff
@@ -39,6 +39,7 @@ attach_all(){
 	    if [ $? -ne 0 ]; then
 	       RC=1
 	       logger "Attempt to attach VDI: ${UUID} failed -- skipping (Error was: ${OUTPUT})"
+	       return $RC
```

Comment:
This could be also implemented with a `break`.

Reply:
Yes. :+1:  Here we neeed to understand whether it is reasonable. I didn't come up with a reason not do this.

Reply:
I think we can remember which static VDI is attached successfully, but to make it simpler, it may not need to do that.